asreview/webapp/src/ProjectComponents/AnalyticsComponents/ProgressDensityChart.js

```diff
@@ -353,10 +356,13 @@ export default function ProgressDensityChart(props) {
                               <Typography variant="subtitle2">
                                 Persistent irrelevant records
                               </Typography>
-                              <Typography variant="body2">
-                                More relevant records might not appear. Refer to
-                                your stopping rules to decide if you want to
-                                continue reviewing.
+                              <Typography
+                                variant="body2"
+                                sx={{ color: "text.secondary" }}
+                              >
+                                Relevant records do not appear. Refer to your
```

Comment:
```suggestion
                                Relevant records no longer appear. Refer to your
```

Reply:
to indicate it is an ongoing process

Reply:
Would "no longer" be true? It is still possible to see relevant records even after a period of irrelevant presence?

Reply:
o yes, indeed... whoops